src/common/formatFamiliesData.ts

```diff
@@ -1,28 +1,77 @@
 import { Person } from "@/components/Form/formFunctions";
 import { Schema } from "@/databaseUtils";
 import { displayList } from "@/common/format";
+import { getChildAgeUsingBirthYearAndMonth, isChildUsingBirthYear } from "@/common/getAgesOfFamily";
+import { getCurrentYear } from "@/common/date";
 
 export interface HouseholdSummary {
     householdSize: string;
     genderBreakdown: string;
+    ageAndGenderOfAdults: string;
     numberOfBabies: string;
     ageAndGenderOfChildren: string;
 }
 
 const getChild = (child: Person): string => {
-    const gender = child.gender === "male" ? "M" : child.gender === "female" ? "F" : "-";
-    return `${child.age} ${gender}`;
+    let gender;
+    switch (child.gender) {
+        case "male":
+            gender = "M";
+            break;
+        case "female":
+            gender = "F";
+            break;
+        case "other":
+            gender = "-";
+            break;
+    }
+
+    const age = getChildAgeUsingBirthYearAndMonth(child.birthYear, child.birthMonth ?? null, true);
+
+    return `${age} ${gender}`;
+};
+
+const getAdult = (adult: Person): string => {
+    let gender;
+    switch (adult.gender) {
+        case "male":
+            gender = "M";
+            break;
+        case "female":
+            gender = "F";
+            break;
+        case "other":
+            gender = "-";
```

Comment:
I think the dash is a bit confusing, maybe just use "O"? Hopefully that's clear enough with others mentioned above it

Reply:
Sure :)